webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7541,18 +7541,20 @@ var userOverride = {
 
             _this.loadDropdownOptions = function () {
                 var enableDropdown = false;
+                var isConvRated = appOptions.oneTimeRating &&
+                    (!KommunicateUI.convRatedTabIds[CURRENT_GROUP_DATA.tabId] == 2);
```

Comment:
@ChannuBS Do not use magic number https://www.pluralsight.com/tech-blog/avoiding-magic-numbers/